.github/workflows/test-chart.yml

```diff
@@ -0,0 +1,79 @@
+name: Lint and Test Charts
+
+on: 
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'helm-chart/**'
+      - '!helm-chart/**/README.md'
+      - '!helm-chart/**/.helmignore'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: "Checkout master branch"
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          path: main
+
+      - name: "Check version is bumped"
```

Comment:
Would make sense to support releasing patches for older versions?
Like if we release a new version 2.0.0 in master (breaking compatibility with 1.x), and some cluster is with 1.5.3, should we support (here) being able to generate the 1.5.4 after 2.0.0?

I guess if we'd need to support that case in future, we could easily "replace" the `exit` here and being a "warning" instead.

Reply:
If that's a common case, (what would be the use case? cuz it strikes to me you would like to bump to 2.0.0 in that case anyway, why staying at 1.5.4?) we can follow the same approach as Bitnami that only checks versions are changed (https://github.com/bitnami/charts/blob/4918597a44c486a0546e9313c46fb2c03904908e/.github/workflows/ci-pipeline.yaml#L57, https://github.com/bitnami/charts/blob/4918597a44c486a0546e9313c46fb2c03904908e/.github/workflows/ci-pipeline.yaml#L65).

In that case, I would like to discuss a branching/tagging strategy as I wouldn't like to have 1.5.4 in master/main and 2.0.0 somewhere lost in the repo.

Reply:
I guess an example situation could be the need to support old versions (because deployments that cannot be upgraded to new) and new versions of the chart do not support old versions of your app (and you need to continue supporting them somehow).